python3-pyslurm.spec

```diff
@@ -0,0 +1,51 @@
+Name:		python3-pyslurm
+Version:	20.02.0
+Release:	1%{?dist}
+Summary:	python3-pyslurm
+
+Group:		Development/Libraries
+License:	GPLv2
+URL:		https://github.com/PySlurm/pyslurm
+
+BuildRequires:	slurm-devel, python3, python3-devel, python3-pip
+Requires:	slurm
+
+%description
+PySLURM is a Python/Cython extension module to the Simple Linux Unified Resource Manager (SLURM) API
+
+%prep
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+export LANGUAGE=en_US.UTF-8
+rm -rf pyslurm
+git clone https://github.com/bikerdanny/pyslurm.git
```

Comment:
Is this RPM spec meant to point to your fork?

Reply:
Sorry, you are right.
Please change this to...
https://github.com/PySlurm/pyslurm.git